clc/modules/object-storage/src/main/java/com/eucalyptus/objectstorage/pipeline/binding/ObjectStorageGETBinding.java

```diff
@@ -88,16 +88,13 @@
     newMap.put(BUCKET + HttpMethod.GET.toString() + ObjectStorageProperties.BucketParameter.marker.toString(), "ListBucket");
```

Comment:
Unrelated to this code change - supported and unsupported operations are a constant. Generate a copy of these maps statically when the class loads instead of creating a new one every single time